pkg/serverless/daemon/daemon.go

```diff
@@ -435,3 +447,17 @@ func (d *Daemon) setTraceTags(tagMap map[string]string) bool {
 	}
 	return false
 }
+
+// IsLambdaLibraryDetected returns if the Lambda Library is in use
+func (d *Daemon) IsLambdaLibraryDetected() bool {
+	d.LambdaLibraryStateLock.RLock()
+	defer d.LambdaLibraryStateLock.RUnlock()
+	return d.LambdaLibraryDetected
```

Comment:
These methods are missing tests. Would be useful to include a test that hits the appropriate endpoint, then checks the return value of this here method.